### ISSUE TITLE
Puzzle floor tiles can be crowbarred ad infinitum

### ISSUE THREAD

**Issue by adamkad1** [tgstation/tgstation#44443]
title, you can crowbar puzzle floor tiles (like from puzzle cube) infinitely, leading to a LOT of metal

**Comment by NeonNik2245** [comment]
Tested that right now - Doesn't work, floor just get deleted, same on other any floor tiles, can close.

https://github.com/user-attachments/assets/538264b7-a17d-4a29-b20d-1a061de8f075

**Comment by NeonNik2245** [comment]
@jlsnow301, can I ask you for help?

**Comment by Sealed101** [comment]
yeah lmao it's been 5+ years, probably got obliterated by some sort of floor tile refactor since the puzzle's tile now doesn't have a `floor_tile` to drop when destroyed/pried up
